manifests/init.pp

```diff
@@ -115,104 +139,59 @@
   Boolean $candlepin_db_ssl = $::katello::params::candlepin_db_ssl,
   Boolean $candlepin_db_ssl_verify = $::katello::params::candlepin_db_ssl_verify,
   Boolean $candlepin_manage_db = $::katello::params::candlepin_manage_db,
+
+  Boolean $enable_candlepin = $::katello::params::enable_candlepin,
+  Boolean $enable_qpid = $::katello::params::enable_qpid,
+  Boolean $enable_qpid_client = $::katello::params::enable_qpid_client,
+  Boolean $enable_pulp = $::katello::params::enable_pulp,
+  Boolean $enable_application = $::katello::params::enable_application,
+
+  String $candlepin_hostname = $::katello::params::candlepin_hostname,
+  String $pulp_hostname = $::katello::params::pulp_hostname,
+  String $qpid_hostname = $::katello::params::qpid_hostname,
+  Stdlib::HTTPUrl $candlepin_url = "https://${candlepin_hostname}:8443/candlepin",
```

Comment:
@Klaas- this may work in pure puppet but with the installer this breaks:

```
 | URI::InvalidURIError: bad URI(is not URI?): https://${candlepin_hostname}:8443/candlepin
```

I'd prefer not to split the hostname and URLs if possible. Shall we try with just the segments exposed and compose the URL variables inside the body?